src/main/java/uk/gov/justice/hmpps/prison/service/InmateService.java

```diff
@@ -164,7 +164,7 @@ public List<InmateBasicDetails> getBasicInmateDetailsForOffenders(final Set<Stri
     }
 
     private boolean isViewAllOffenders() {
-        return authenticationFacade.isOverrideRole("SYSTEM_USER", "GLOBAL_SEARCH", "VIEW_PRISONER_DATA");
```

Comment:
isViewAllOffenders() has been called by 6 clients in the last year.  5 do not have the SYSTEM_USER role and the 6th client (categorisation-tool) passes through the USER details.